data/proposals.js

```diff
@@ -525,5 +525,14 @@ module.exports = [
       "gstepniewski"
     ],
     "wiki": "Wilfred (U.S. TV series)"
-  }
+  },
+  {
+    "title": "Over the Garden Wall",
```

Comment:
Please keep the alphabetical order. You can always add a new TV show with commadline tool, by `grunt proposal --id="imdbid"` and then you'll probably need to change your gh username in the `data/proposals.js` probably, I'm not sure if thats implemented.
In your case it will be probably `grunt proposal --id="tt3718778'`.

Thanks for the contribution!